packages/data-point/lib/reducer-types/reducer-list/factory.js

```diff
@@ -79,12 +78,11 @@ module.exports.parse = parse
  * @param {Array} source
  * @return {reducer}
  */
-function create (createReducer, source = []) {
```

Comment:
I was getting < 100% branch coverage because of this default, which is redundant because there's a default added in the `parse` function as well